app/routes/screens/race/actionsCreator.js

```diff
@@ -0,0 +1,8 @@
+import { bindActionCreators } from "redux";
+import * as geoloactionActions from "../../../store/actions/geolocation";
+
+const actionsCreator = dispatch => ({
+  geoloactionActionsCreators: bindActionCreators(geoloactionActions, dispatch)
```

Comment:
this was the most important part to be reviewed! Not sure how to do that in elegant way!